### PR TITLE
Change description in constructor to a named parameter (as it is mentioned later on that page)

### PR DESCRIPTION
**Pull request by ViktorVoloshko**
_Description of what this PR is changing or adding, and why:_ Later on description says:
"You can see this demonstrated in the constructor for the Package class. It takes two required, positional arguments and **one optional, named argument**."

However, code had 3 required positional arguments.

## Presubmit checklist

- [x] This PR is marked as draft with an explanation if not meant to land until a future stable release.
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/content/get-started/fundamentals/dart.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ class Package {
   final String latestVersion; 
   final String? description;
 
-  Package(this.name, this.latestVersion, this.description);
+  Package(this.name, this.latestVersion, {this.description});
```
Comment (sfshaza2) [inline src/content/get-started/fundamentals/dart.md:51]: I don't think this is the correct solution. Shouldn't an optional named parameter be put in square brackets?

@ericwindmill

Comment (ericwindmill) [reply src/content/get-started/fundamentals/dart.md:51]: This is correct, named parameters use curly braces

**Review by sfshaza2 — APPROVED** [review]
ok, thx! lgtm

**Comment by sfshaza2** [comment]
@ViktorVoloshko, if you can remove this from Draft, I can land.

**Comment by ViktorVoloshko** [comment]
@sfshaza2 done.

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**